projects/plugins/jetpack/class.jetpack-cli.php

```diff
@@ -2010,6 +2010,7 @@ public function block( $args, $assoc_args ) {
 			"$path/$slug.php"     => self::render_block_file(
 				'block-register-php',
 				array(
+					'nextVersion'      => '$$next-version$$',
```

Comment:
Have to disguise it somehow so replace-next-version-tag.sh doesn't see it here. There are various ways you might do that, e.g.
```suggestion
					'nextVersion'      => "\x24\x24next-version$$", // Escapes to hide the string from tools/replace-next-version-tag.sh
```